### PR TITLE
Modal

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: components/modal.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    position: fixed;
+    display: none;
+}
+.show{
+    display: block;
+}
+.content-modal{
+    background: white;
+    padding: 10px 20px 20px 20px;
+    width: 300px;
```
Comment (Manpreet-hub) [inline components/modal.css]: Suggestion: Try to use here relative unit.

File: components/modal.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+.container-modal{
+    width: 100vw;
+    height: 100vh;
+    box-sizing: border-box;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    padding-top: 30vh;
+    background: rgb(0,0,0);
+    background: #00000066;
```
Comment (mansi2011) [inline components/modal.css]: you have added color value directly it would be good to have a variable for this

**Review by Manpreet-hub — COMMENTED** [review]
Rest all looks good!

File: components/modal.css [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    position: fixed;
+    display: none;
+}
+.show{
+    display: block;
+}
+.content-modal{
+    background: white;
```
Comment (mansi2011) [inline components/modal.css:18]: same again

File: components/modal.css [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    padding: 10px 20px 20px 20px;
+    width: 300px;
+    height: max-content;
+    border-radius: 4px;
+    box-shadow: 3px 3px 20px 10px #00000023;
```
Comment (mansi2011) [inline components/modal.css]: also here

**Review by mansi2011 — COMMENTED** [review]
code looks code suggested some point

File: components/modal.css (hunk 4 above)
Comment (shivanigangadharan) [reply components/modal.css]: done [6ed187c](https://github.com/shivanigangadharan/style-ui/pull/18/commits/6ed187cf3eacb55e1ac50168a806ce767dc7c359)

File: components/modal.css (hunk 3 above)
Comment (shivanigangadharan) [reply components/modal.css:18]: done [6ed187c](https://github.com/shivanigangadharan/style-ui/pull/18/commits/6ed187cf3eacb55e1ac50168a806ce767dc7c359)

File: components/modal.css (hunk 1 above)
Comment (shivanigangadharan) [reply components/modal.css]: done in commit [6ed187c](https://github.com/shivanigangadharan/style-ui/pull/18/commits/6ed187cf3eacb55e1ac50168a806ce767dc7c359) using clamp

File: components/modal.css (hunk 2 above)
Comment (shivanigangadharan) [reply components/modal.css]: done in commit [6ed187c](https://github.com/shivanigangadharan/style-ui/pull/18/commits/6ed187cf3eacb55e1ac50168a806ce767dc7c359)

**Pull request merged**